server/db/seeds/development/seed-admins-table.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  async seed(knex) {
+    // Deletes all existing entries
+    await knex('admins').del()
+
+    // Insert seed entries
+    await knex('admins').insert([
+      {
+        id: 1,
+        first_name: 'John',
+        last_name: 'Doe',
+        email: 'johndoe@gmail.com',
+        password:
```

Comment:
For the purpose of development, you might want to put something more generic in the seed, like `password`.